CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 1.2.4 (2018-03-14)
+
+### Fixes
+
+- Error responses are now user errors by default, rather than internal errors [weaveworks/flux#942](https://github.com/weaveworks/flux/pull/942)
+- CLI help examples updated with new resource ID format [weaveworks/flux#945](https://github.com/weaveworks/flux/pull/945)
+- Fix a panic caused by accessing a `nil` map when logging events [weaveworks/flux#975](https://github.com/weaveworks/flux/pull/975)
+- Properly support multi-line lock messages [weaveworks/flux#978](https://github.com/weaveworks/flux/pull/978)
+- Ignore Helm charts when looking for Kubernetes manifests [weaveworks/flux#993](https://github.com/weaveworks/flux/pull/9993)
+
+### Improvements
+
+- Enable pprof [weaveworks/flux#927](https://github.com/weaveworks/flux/pull/927/files)
+- Use a Kubernetes serviceAccount when deploying Flux standalone [weaveworks/flux#972](https://github.com/weaveworks/flux/pull/972)
+- Ensure at-least-once delivery of events to Weave Cloud [weaveworks/flux#973](https://github.com/weaveworks/flux/pull/973)
+- Include resource sync errors when logging a sync event [weaveworks/flux#970](https://github.com/weaveworks/flux/pull/970)
+
```

Comment:
This release also contains the alpha stage of helm integration. Is it worth mentioning? Perhaps the CHANGELOG only refers to direct flux agent changes :thinking: In which case we should have another CHANGELOG for the helm-operator.

Reply:
Let's have another section, titled `Experimental`, and linky the helm operator and its instructions there.

Reply:
Not sure if we are talking about the same. I mean implementation changes, bug fixes and new features for helm-operator that are delivered in a release. If your "instructions" mean that too then we are on the same page.

Reply:
> This release also contains the alpha stage of helm integration. Is it worth mentioning? 

The set of commits between `1.2.3` and `1.2.4` include the addition of the helm-operator, but the flux release itself is independent of that. #1004 captures the current situation and attempts to describe how we resolve it.

Reply:
> we should have another CHANGELOG for the helm-operator.

👍